app/helpers/arclight_helper.rb

```diff
@@ -34,7 +34,7 @@ def collection_count
   end
 
   def repositories_active?
-    controller_name == 'repositories'
+    controller_name == 'repositories' && action_name == 'index'
```

Comment:
This seems to turn off the `active` ability of the link here:
![screen shot 2017-05-23 at 6 38 45 pm](https://cloud.githubusercontent.com/assets/1656824/26379409/22b4af46-3fe7-11e7-8298-b0964507477d.png)
Should that still be active?